src/main/java/net/snowflake/client/jdbc/SnowflakeDatabaseMetaData.java

```diff
@@ -1188,11 +1188,7 @@ public ResultSet getProcedureColumns(
           nextRow[3] = paramNames[i]; // column/parameter name
           // column type
           if (i == 0) {
-            if (paramTypes[i].substring(0, 5).equalsIgnoreCase("table")) {
-              nextRow[4] = procedureColumnOut;
-            } else {
-              nextRow[4] = procedureColumnReturn;
-            }
+            nextRow[4] = procedureColumnReturn;
```

Comment:
How do we get actual OUT data type? Try a proc with return value and out parameter.

Reply:
@sfc-gh-igarish does the Snowflake stored procedure support output parameters? I don't see any syntax for OUT/IN_OUT only IN and return value in the documentation.